bin/sidekiq_wrapper

```diff
@@ -5,17 +5,20 @@ require_relative '../lib/environment_light.rb'
 
 RAILS_ROOT = File.expand_path('../..', __FILE__).freeze
 SIDEKIQ_EXECUTABLE = File.join(RAILS_ROOT, 'bin/sidekiq').freeze
-SIDEKIQ_BASE_ARGUMENTS =
-  %W(#{SIDEKIQ_EXECUTABLE}
-     --environment #{ENV['RAILS_ENV'] || 'production'}).freeze
 
 # Parse options
 def parse_options
-  options = {workdir: Dir.pwd,
+  options = {environment: 'production',
+             workdir: Dir.pwd,
              timeout: 15}
   OptionParser.new do |opts|
     opts.banner = "Usage: #{File.basename(__FILE__)} [options]"
 
+    opts.on('-eMANDATORY', '--environment=MANDATORY',
+            "Set Rails environment (default: #{options[:environment]})") do |e|
+      options[:environment] = e
+    end
+
     opts.on('-wMANDATORY', '--workdir=MANDATORY',
             'Set work directory (default: CWD)') do |w|
```

Comment:
BTW: dict lookup __MANDATORY__!

Also as a general rule of thumb: In synopsis, usage infos the option value names should indicate its "type", e.g. -w *directory*, -t *seconds*. The description should just include '(mandatory)' or '(optional)' if needed. Usually it is not, because the default value should be mentioned and than it is clear, that it is 'optional'. For arg-less options the name _option_ indicates, that it is _option_al and thus doesn't need to be explicitly mentioned. So only in very rare cases or combination of options may require, that 'mandatory' appears in documentation.